tests/unit/test_wagtail_hooks.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from wagtail.admin import widgets
+from wagtail.core import hooks
+
+from tests.factories.pages import ArticleFactory
+from wagtailtrans.models import TranslatablePage
+from wagtailtrans.wagtail_hooks import edit_in_language_button
+
+
+@pytest.mark.django_db
+class TestWagtailHooks:
+    def setup(self):
```

Comment:
We don't need the 2 pages created in the setup for each test, so lets just create the page we need in the test itself